pypdf/_writer.py

```diff
@@ -862,6 +862,7 @@ def _update_text_field(self, field: DictionaryObject) -> None:
         # Extract font information
         da = cast(str, field[AA.DA])
         font_properties = da.replace("\n", " ").replace("\r", " ").split(" ")
+        font_properties = [x for x in font_properties if x != ""]
```

Comment:
@pubpub-zz  Is this an equivalent solution?

```suggestion
        font_properties = da.replace("\n", " ").replace("\r", " ").replace(" ", "").split(" ")
```

Reply:
Oh, never mind, it's not equivalent